Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -254,7 +254,7 @@ + (void)initialize
     // Redirect NSLogs to files only if we are not debugging
     if (!isatty(STDERR_FILENO))
     {
-        [MXLogger redirectNSLogToFiles:YES numberOfFiles:50];
+        [MXLogger redirectNSLogToFiles:YES numberOfFiles:50 sizeLimit:(100 * 1024 * 1024)];
```

Comment:
Shouldn't we have constants so we can change those values easily?

Reply:
Do you know a SDK constant for 1MB?